docs/usecases/streaming.md

```diff
@@ -3,4 +3,105 @@ id: usecases_streaming
 title:  "Streaming"
```

Comment:
@almendar How about we move the excellent breakdown you've written to the "Advanced" section?